docker/candlepin-base-cs9/setup-devel-env.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+#
+# Sets a system up for a candlepin development environment (minus a db,
+# handled separately), and an initial clone of candlepin.
+
+set -ve
+
+source /root/dockerlib.sh
+
+export JAVA_VERSION=11
+export JAVA_HOME=/usr/lib/jvm/java-$JAVA_VERSION
+
+# Install & configure dev environment
+dnf install -y epel-release
+
+PACKAGES=(
+    createrepo_c
+    expect
+    gettext
+    git
+    hostname
+    java-$JAVA_VERSION-openjdk-devel
+    jss-5.0.3-1.el9 # last jss package that support Java version 11
+    mariadb
+    openssl
+    pki-servlet-engine
+    postgresql
+    procps
+    python3-pip
+    python3-pyyaml
+    python3-requests
+    python3-libxml2
+    python-unversioned-command
+    rpm-build
+    rpm-sign
+    rsyslog
+    wget
+    which
+    zlib
+    zlib-devel
+)
+
+dnf install -y ${PACKAGES[@]}
+
+# Setup for autoconf:
+mkdir /etc/candlepin
+echo "# AUTOGENERATED" > /etc/candlepin/candlepin.conf
+
+cat > /root/.bashrc <<BASHRC
+if [ -f /etc/bashrc ]; then
+  . /etc/bashrc
+fi
+
+export HOME=/root
+export JAVA_HOME=/usr/lib/jvm/java-$JAVA_VERSION
+BASHRC
+
+git clone https://github.com/candlepin/candlepin.git /candlepin
+cd /candlepin
+
+# Setup and install rvm, ruby and pals
+gpg2 --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+# turning off verbose mode, rvm is nuts with this
+set +v
+curl -O https://raw.githubusercontent.com/rvm/rvm/master/binscripts/rvm-installer
+curl -O https://raw.githubusercontent.com/rvm/rvm/master/binscripts/rvm-installer.asc
+gpg2 --verify rvm-installer.asc && bash rvm-installer stable
+source /etc/profile.d/rvm.sh || true
+
+rvm pkg install openssl
+rvm install ruby-2.5.3 --with-openssl-dir=/usr/local/rvm/usr
+rvm use --default 2.5.3
+set -v
+
+# Install all ruby deps
+gem install bundler -v 1.16.1
+bundle install --without=proton --retry=5 --verbose --full-index
```

Comment:
Lets remove all this. We no longer need ruby

Reply:
done